.gitignore

```diff
@@ -10,6 +10,7 @@ distribute-*
 env
 local
 .tox
+pip-wheel-metadata
```

Comment:
This entry *should* not be necessary anymore - pip has been changed in pypa/pip#6325 to use a temporary directory instead of one in the project directory, and this change is included in pip 19.3 and later (see the [changelog](https://pip.pypa.io/en/stable/news/), under 19.3, under Bug Fixes).

Reply:
I added it because it suddenly appeared in my Rubicon folder after some testing; I'm guessing the environment I was in had an older version of pip. If it's not required on a modern pip, then I agree it can be removed.